lib/telegraph.rb

```diff
@@ -0,0 +1,34 @@
+require 'faraday'
+require 'nokogiri'
+require 'json'
+require 'singleton'
+require_relative 'telegraph/node_helper'
+require_relative 'telegraph/connection'
+require_relative 'telegraph/node_element'
+require_relative 'telegraph/client'
+require_relative 'telegraph/page_list'
+require_relative 'telegraph/page_views'
+require_relative 'telegraph/account'
+require_relative 'telegraph/page'
+require_relative 'telegraph/configuration'
+
+module Telegraph
+  def self.setup(token)
+    configuration.key = token
+    @client = Client.new(configuration.key)
+  end
```

Comment:
What are the main use cases for "API wrapper" library?
* create bots (probably several of them for the different cases);
* integrate telegraph to some (probably multi-user) website;
* post on user's behalf (from some service with multiple users).

For most of the reasonable cases, library user does NOT want global config, but something like:
```ruby
client = Telegraph::Client.new(some_current_user.token)
```
...and, in general, it is a good taste to avoid globals (and global configurations) in libraries.